sqlutil/sqlutil.go

```diff
@@ -0,0 +1,48 @@
+package sqlutil
```

Comment:
Should this be in the `util` directory? `util/sql/sql.go`

Reply:
Yes, I'll move it.